packages/lesswrong/components/common/Header.tsx

```diff
@@ -170,13 +222,16 @@ const Header = ({
   toggleStandaloneNavigation,
   stayAtTop=false,
   searchResultsArea,
+  backgroundColor,
   classes,
 }: {
   standaloneNavigationPresent: boolean,
   sidebarHidden: boolean,
   toggleStandaloneNavigation: ()=>void,
   stayAtTop?: boolean,
   searchResultsArea: React.RefObject<HTMLDivElement>,
+  // CSS var corresponding to the background color you want to apply (see also appBarDarkBackground above)
+  backgroundColor?: string,
```

Comment:
Since we've done this a couple times, I'm guessing Agnes will want to do this more in the future, so I made a reasonable default for header styles with a single dark background

Reply:
I think this works even if you don't use a css var, so I'm just going to leave it as-is